miqi/runtime/file_handlers.py

```
@@ -1057,3 +1058,92 @@ async def files_accept_handler(
+        }
+    return "not_found", {"resolvedPath": str(resolved)}
+
+
+async def files_check_many_handler(
+    request_id: str,
+    params: dict[str, Any],
+    client_id: str,
+    session_id: str | None,
+    registry: Any,
+) -> dict[str, Any]:
+    """#790: 批量校验资产路径可达性（渲染资产卡片时按需调用）。
```

Ruff (0.16.2)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)